README.md

```diff
@@ -19,6 +19,34 @@ Run `npm run build:all` from the root directory to build each application at onc
 
 Add and configure multiple `.env` files accordingly using `.env.example` templates .
 
+# Environment variables
+
+Create .env files in backend and frontend application directory (next to .env.example)
+
+### Backend
+
+see [.env.example](applications/backend/.env.example)
+
+| variable             | description                                                                                           |
+| -------------------- | ----------------------------------------------------------------------------------------------------- |
+| CLIENT_ID_SECRET     | a controlpanel related secret to generate a jwt token to create an access token for the rest services |
```

Comment:
as a noobie who doesn't know anything it would be great to know if these env vars need a certain value or if you can just set a random value for local development

Reply:
in this case yes (even adding only one number or character), it depends on the framework but basically any long, complicated hash does the job